mantidproject/mantid
# Fix reading dead time data in Muon interface

The cache in muon analysis is now cleared when the data is updated (change to the dead time correction, first good data and start time). 

**To test:**


In muon analysis make sure that when the screen is generated "None" is selected for the dead time correction. Load some data and then change the dead time correction and the plot should be updated. 
Fixes #18950 .



---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
